pom.xml

```diff
@@ -1155,6 +1157,55 @@
 				</plugins>
 			</build>
 		</profile>
+
+		<profile>
+			<id>deb</id>
+			<build>
+				<plugins>
+					<plugin>
+						<groupId>org.vafer</groupId>
+						<artifactId>jdeb</artifactId>
+						<version>1.3</version>
+						<executions>
+							<execution>
+								<goals>
+									<goal>jdeb</goal>
+								</goals>
+								<configuration>
+									<skipPOMs>true</skipPOMs>
+									<deb>${project.build.directory}/${project.artifactId}-${project.version}.deb</deb>
+									<dataSet>
+										<data>
+											<src>${project.build.directory}/jmxtrans-${project.version}-dist.tar.gz</src>
```

Comment:
.deb deploys more or less correctly (at least JmxTrans is starting without error). `apt-get remove` still does not seem to work.

As this is already a working increment over the current situation (no .deb created at all), I would tend to merge it as-is. But I'd like to have your feedback before.